static/app/views/releases/list/releasesPromo.tsx

```diff
@@ -117,60 +104,103 @@ const ReleasesPromo = ({organization, project}: Props) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
-  const trackQuickstartCopy = useCallback(() => {
-    trackAdvancedAnalyticsEvent('releases.quickstart_copied', {
-      organization,
-      project_id: project.id,
-    });
-  }, [organization, project]);
+  const generateToken = useCallback(
+    async (sentryAppSlug: string) => {
+      const newToken = await api.requestPromise(
+        `/sentry-apps/${sentryAppSlug}/api-tokens/`,
+        {
+          method: 'POST',
+        }
+      );
+      return newToken.token;
+    },
+    [api]
+  );
 
-  const trackQuickstartCreatedIntegration = useCallback(
-    (integration: SentryApp) => {
-      trackAdvancedAnalyticsEvent('releases.quickstart_create_integration.success', {
-        organization,
-        project_id: project.id,
-        integration_uuid: integration.uuid,
-      });
+  const fetchToken = useCallback(
+    async sentryAppSlug => {
+      const tokens = await api.requestPromise(
+        `/sentry-apps/${sentryAppSlug}/api-tokens/`
+      );
+      if (!tokens.length) {
+        const newToken = await generateToken(sentryAppSlug);
+        return setToken(newToken);
+      }
+      return setToken(tokens[0].token);
     },
-    [organization, project]
+    [api, generateToken]
   );
 
-  const trackCreateIntegrationModalClose = useCallback(() => {
-    trackAdvancedAnalyticsEvent('releases.quickstart_create_integration_modal.close', {
-      organization,
-      project_id: project.id,
+  const createReleaseIntegration = useCallback(async () => {
+    const newIntegration = await api.requestPromise('/sentry-apps/', {
+      method: 'POST',
+      data: {
+        name: `${project.name} Release Integration`,
```

Comment:
The integration isn't specific to a project, it's organization wide. What do you think of putting the org name here (and in the description) instead?